ui/sections/network/inspectors/network-interface.reel/interface.reel/interface.js

```diff
@@ -4,4 +4,15 @@ var Component = require("montage/ui/component").Component;
  * @class Interface
  * @extends Component
  */
-exports.Interface = Component.specialize();
+exports.Interface = Component.specialize({
+    active_media_type: {
+        value: null
+    },
+
+    templateDidLoad: {
+        value: function () {
+            var self = this;
+            self.active_media_type = this.object.status.active_media_type.concat(" ", this.object.status.active_media_subtype);
```

Comment:
`this.object` is not guaranteed to be set when `templateDidLoad` is called. Moreover, `templateDidLoad` is only called once when the component is first loaded, so it won't reflect object changes.
This code should be moved to `enterDocument`.